README.md

```diff
@@ -61,7 +61,10 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
-      timeout_seconds:  1
+      timeout_seconds:  1,
+
+      # lambda determining what response is considered a failure 
+      error_response: lambda { |response| response.status >= 500 }
```

Comment:
I am confused, better sync-up on monday (I cannot find anymore the code where I setup the middlewares in the client).